Gemfile.lock

```diff
@@ -487,7 +487,7 @@ GEM
     websocket-driver (0.7.2)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
-    wkhtmltopdf-binary (0.12.5.4)
+    wkhtmltopdf-binary-edge-alpine (0.12.5.0)
```

Comment:
@duncanjbrown in #2621 , we could mount just the `/app` folder instead of the root folder containing the gem files?
Most files requiring content change should be inside the /app folder?